src/app/modules/payments/stripe.utils.ts

```diff
@@ -329,3 +331,48 @@ export const mapRouteErr = (error: ApplicationError) => {
       }
   }
 }
+
+/**
+ * Converts receipt sourced from Stripe into an invoice format
+ * @param receiptHtmlSource
+ */
+export const convertToInvoiceForrmat = (receiptHtmlSource: string) => {
+  // handle special characters in addresses
+  const ADDRESS = encode('8 Jurong Town Hall Road; Singapore 609434')
+  const GST_REG_NO = encode('M91234567X')
+
+  fs.writeFileSync('./' + Date.now(), receiptHtmlSource)
```

Comment:
I think this is not needed 😅 . It was there in the script so I could see the receipt files locally, but it's meaningless to write to disk on the servers.

Reply:
Oops. Forgot to remove this. I'm using it to debug too 😂